crd/apis/gcpfirewall/v1beta1/types.go

```diff
@@ -200,6 +200,9 @@ const (
 	// FirewallRuleReasonPending is used when the firewall rule is not synced to
 	// GCP and enforced yet.
 	FirewallRuleReasonPending FirewallRuleConditionReason = "Pending"
+
+	// FirewallRuleReasonXPNError is used when the controller does not have the perssion to configure firewalls in the shared VPC
```

Comment:
Let's call it a XPNPermissionError? Since we're specific, we might as well go all the way?

Reply:
Done.